components/EmptyState/EmptyStateInactive.js

```diff
@@ -57,14 +57,14 @@ class EmptyStateInactive extends React.Component {
 
   startService(e) {
     if (!e || e.button !== 0) return;
-    let argv;
+    let script;
     if (this.state.enableService) {
-      argv = ["systemctl", "enable", "--now", "osbuild-composer.socket"];
+      script = "systemctl enable --now osbuild-composer.socket && systemctl enable --now osbuild-worker@1";
```

Comment:
Is it really needed to `enable --now osbuild-worker@1`? Doesn't it mean that osbuild-worker is always running in the background even when osbuild-composer isn't running?

Reply:
You are correct. We only need to start the socket. Given David's comment here https://github.com/osbuild/osbuild-composer/pull/330#discussion_r393570358 I am now wondering if we should enable the socket or just start it.